packages/multiple-choice/configure/src/defaults.js

```diff
@@ -17,12 +17,16 @@ export default {
     limitChoicesNumber: true,
     teacherInstructionsEnabled: true,
     studentInstructionsEnabled: true,
-    verticalMode: true
+    choicesLayout: 'vertical',
+    gridColumns: 2
   },
   configuration: {
-    verticalMode: {
+    choicesLayout: {
```

Comment:
Since you're removing an existing property, I think it would be better to have a breaking change.